ui/droppable.js

```diff
@@ -33,6 +33,7 @@ $.widget( "ui.droppable", {
 		accept: "*",
 		activeClass: false,
 		addClasses: true,
+		classes: {},
 		greedy: false,
 		hoverClass: false,
```

Comment:
Should probably address `activeClass` and `hoverClass`, but we may want to just want for the rewrite.

@jzaefferer @mikesherov thoughts?